install_config/install/docker_registry.adoc

```diff
@@ -165,6 +166,10 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
+[IMPORTANT]
+====
+The docker registry pod run as user *1001*. This user needs to be able to write to the host directory. You may need to change directory ownership to user ID *1001* with this command: `sudo chown 1001:root <path>`
```

Comment:
s/docker/Docker/

s/run/runs/

s/needs to/must/

Try putting the command in a code block (with a `$` prompt):

```
with this command:

----
$ sudo chown 1001:root <path>
----
====
```